src/converge.js

```diff
@@ -1,22 +1,22 @@
+var _curry2 = require('./internal/_curry2');
 var _map = require('./internal/_map');
-var _slice = require('./internal/_slice');
 var curryN = require('./curryN');
 var pluck = require('./pluck');
 
 
 /**
- * Accepts at least three functions and returns a new function. When invoked, this new
- * function will invoke the first function, `after`, passing as its arguments the
- * results of invoking the subsequent functions with whatever arguments are passed to
- * the new function.
+ * Accepts a converging function and a list of branching functions and returns a new function.
+ * When invoked, this new function is applied to some arguments, each branching
+ * function is applied to those same arguments. The results of each branching
+ * function are passed as arguments to the converging function to produce the return value.
  *
  * @func
  * @memberOf R
  * @category Function
- * @sig (x1 -> x2 -> ... -> z) -> ((a -> b -> ... -> x1), (a -> b -> ... -> x2), ...) -> (a -> b -> ... -> z)
+ * @sig (x1 -> x2 -> ... -> z) -> ([(a -> b -> ... -> x1), (a -> b -> ... -> x2), ...]) -> (a -> b -> ... -> z)
```

Comment:
Now that we have `[` and `]` we could remove the surrounding parens (i.e. `[...]` rather than `([...])`).